core/services/ocr2/delegate.go

```diff
@@ -1008,7 +1011,25 @@ func (d *Delegate) newServicesOCR2Keepers21(
 		services.UpkeepStateStore(),
 		services.TransmitEventProvider(),
 		pluginService,
-	}, nil
+	}
+
+	if cfg.CaptureAutomationCustomTelemetry {
```

Comment:
this comment got lost in updates https://github.com/smartcontractkit/chainlink/pull/9927#discussion_r1336042509, could you point this to toml config overridable via job config